src/nodes/linkBlock.ts

```diff
@@ -0,0 +1,75 @@
+import { MyNodeSpec, NodeGroups } from './types';
+import { MdFormatSerialize, TexFormatSerialize } from '../serialize/types';
+import { StaticPhrasingContent, LinkBlock } from '../spec';
+
+export interface Attrs {
+  title: string;
+  description: string;
+  url: string;
+}
+
+const link_block: MyNodeSpec<Attrs, LinkBlock> = {
+  attrs: {
+    url: { default: '' },
+    title: { default: '' },
+    description: { default: '' },
+  },
+  group: NodeGroups.top,
+  content: `${NodeGroups.text}*`,
+  parseDOM: [
+    {
+      tag: 'div[title][block-url]',
```

Comment:
Can we target this with a specific class?

As written this is also not consistent with the [data-url] below! I think using a class is easiest here (the aside has some similar things I think...?)